clang/test/Preprocessor/cheri-features.c

```diff
@@ -25,7 +25,7 @@ void* __capability x = 0;
 
 #if defined(__CHERI__)
```

Comment:
Could we add a test for the new hybrid macro here? And maybe update the RUN lines for RISC-V instead of MIPS? But that can be done in a future commit

Reply:
I've added the hybrid macro, it seems better to do the MIPS->RISC-V conversion in one go (the compiler command names also don't make sense to me)